salt/states/powerdns.py

```diff
@@ -0,0 +1,118 @@
+"""
+Create, modify and delete PowerDNS zone
+"""
+
+import logging
+
+from salt.exceptions import SaltInvocationError
+
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
log = logging.getLogger(__name__)
```
Please name the logging handler instance `log`, not `logger`.

Reply:
Fixed